### PR TITLE
Add `exporters` extra, move setup.cfg to pyproject.toml

### PR DESCRIPTION
**Pull request by henriquegemignani**
Currently not having the `exporters` extra causes issues if you go deep into some game's UI, but at least a lot of things work.

### CODE REVIEW

**Comment by Miepee** [comment]
can you give examples for "causes issues if you go deep into some game's UI"?

**Comment by henriquegemignani** [comment]
Cave Story and Super Metroid immediatelly fails and shows no UI. I didn't investigate SM, but CS uses the patcher to wrap some strings to display at hints and others.

Prime 1 had the same, but I fixed already moving the imports to when you export.
Attempting to export a game causes errors. It's expected, but these aren't handled very well.

File: randovania/games/prime2/layout/echoes_user_preferences.py [hunk 1]
```diff
@@ -1,12 +1,43 @@
...
 
 import dataclasses
-
-from open_prime_rando.dol_patching.echoes.user_preferences import OprEchoesUserPreferences
+import enum
 
 from randovania.bitpacking.json_dataclass import JsonDataclass
 
 
+def _int_field(default: int, min_value: int, max_value: int, display_as_percentage: bool = True):
```
Comment (Miepee) [inline randovania/games/prime2/layout/echoes_user_preferences.py:9]: mypy complains here.

File: randovania/games/prime2/layout/echoes_user_preferences.py [hunk 2]
```diff
@@ -1,12 +1,43 @@
...
+    sfx_volume: int = _int_field(0x69, 0x00, 0x69)
+    music_volume: int = _int_field(0x4F, 0x00, 0x69)
+    hud_alpha: int = _int_field(0xFF, 0x00, 0xFF)
+    helmet_alpha: int = _int_field(0xFF, 0x00, 0xFF)
+    hud_lag: bool = True
+    invert_y_axis: bool = False
+    rumble: bool = True
+    hint_system: bool = False
+
+    def __post_init__(self):
```
Comment (Miepee) [inline randovania/games/prime2/layout/echoes_user_preferences.py:37]: mypy complains here

File: randovania/games/prime1/exporter/game_exporter.py
Comment (ThanatosGit) [inline randovania/games/prime1/exporter/game_exporter.py:1]: Three untyped libraries :(

**Review by ThanatosGit — APPROVED** [review]

**Pull request merged**